devices.js

```diff
@@ -11993,6 +11993,15 @@ const devices = [
         },
         exposes: [e.temperature(), e.humidity(), e.battery()],
     },
+    {
+        zigbeeModel: ['3AFE010104020028'],
+        model: 'TW-S1',
+        description: 'Photoelectric smoke detector',
+        vendor: 'Konke',
+        fromZigbee: [fz.ias_smoke_alarm_1],
+        toZigbee: [],
+        exposes: [e.smoke()],
```

Comment:
Also add `e.battery_low(), e.tamper()` here, if it works battery low should be false (assuming your battery is new), tamper can be triggered by removing the back cover probably.

Reply:
Done. batter_low is false, and the battery is new. I wasn't able to trigger the tamper signal by removing the back cover though. The device looks extremely simple and I don't know how it could detect that.

Reply:
Thanks, removed the tamper stuff.